src/OrchardCore/OrchardCore.Filters.Core/OrchardCore.Filters.Core.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>OrchardCore.Filters.Core</RootNamespace>
+    <!-- NuGet properties-->
+    <Title>OrchardCore Filters Core</Title>
+    <Description>$(OCCMSDescription)
+    
+    Core implementation for core Parsers and Filters module.</Description>
+    <PackageTags>$(PackageTags) OrchardCoreCMS</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.Infrastructure.Abstractions\OrchardCore.Infrastructure.Abstractions.csproj" />
```

Comment:
What do we need a reference to `Infrastructure` for ?

I would have thought it was just `IClock` which is in `OrchardCore.Abstractions`

but could be wrong, so can you check it out and let me know

Reply:
no, you are right. replaced with OrchardCore.Abstractions